lib/_tls_wrap.js

```diff
@@ -1412,6 +1412,8 @@ Server.prototype.addContext = function(servername, context) {
   if (!servername) {
     throw new ERR_TLS_REQUIRED_SERVER_NAME();
   }
+//https://github.com/nodejs/node/issues/34110
+servername=servername.toLowerCase()
```

Comment:
```suggestion
servername = servername.toLowerCase()
```

The `Fixes: https://github.com/nodejs/node/issues/34110` bit should be in the commit message body, not as a comment.

Also, be sure to run `make lint` (`vcbuild lint` on Windows) to make sure the changes pass linting